src/Response.php

```diff
@@ -29,12 +29,19 @@ class Response
 {
     public $responseCode;
     public $json;
+    public $data;
     public $errorMessage;
 
     public function __construct($curlHandle, $curlResult)
     {
         $this->responseCode = curl_getinfo($curlHandle, CURLINFO_HTTP_CODE);
-        $this->json = json_decode($curlResult, true);
+
+        if ('application/json' === curl_getinfo($curlHandle, CURLINFO_CONTENT_TYPE)) {
```

Comment:
I think we can improve this condition in the _construct
- make an external function for this condition
- avoid else and use if in second time
- do we need to use a data property ? It ugly but do we keep json property for both response ?

Reply:
Just rename data to responseFile